packages/web-frontend/config/webpack.config.dev.js

```diff
@@ -161,6 +161,7 @@ module.exports = {
           // It enables caching results in ./node_modules/.cache/babel-loader/
           // directory for faster rebuilds.
           cacheDirectory: true,
+          presets: ['react-app'],
```

Comment:
In the past, we relied on `.babelrc` configuration to be picked up by `babel-loader` in our webpack configuration. This causes some issues with the new `cypress:generate-config` script:

`babel-node cypress/scripts/generateConfig --config-file '../../babel.config.json'`

In this script, we just want to use the specified `config-file` and not include `.babelrc`. Unfortunately, `.babelrc` is picked up automatically by babel.

A solution would be using a `babelrc.json` file, which doesn't get picked up automatically. But then this file isn't picked up by babel-loader 🤷 

My theory is that this happens because `.babelrc` is the (legacy, but still supported) babel configuration file, and `babelrc.json` is a new config file introduced in Babel 7. Apparently `babel-loader` hasn't caught up with this change.

Reply:
Well great work on finding a solution!